src/backer/backering.py

```diff
@@ -299,27 +302,29 @@ def on_post(self, req, rep):
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
 
-        backer_identifiers = serder.ked["b"]
+        # Should check when create identifiers only
+        if serder.ked["t"] == "icp":
```

Comment:
Firstly, we can use the `Ilks` object from keripy instead of hardcoding "icp". But we need more than this.

We need to allow: `icp='icp', rot='rot', ixn='ixn', dip='dip', drt='drt',`

Reply:
Yep, using Ilks.icp instead of 'icp'.
>We need to allow: icp='icp', rot='rot', ixn='ixn', dip='dip', drt='drt'
I change those but can you re-confirm?
I see that Leo want to validate the REGISTRAR_SEAL_SAID for only icp event?

Reply:
Good point. You could also actually start with witnesses for `icp` and then use a `rot` to change to cardano-backer. So we'll need to handle this. But maybe in another story - for now can you just allow each message?

Reply:
I see that we add multiple msg here:
```
msg = bytearray(serder.raw)
msg.extend(cr.attachments.encode("utf-8"))
self.rxbs.extend(msg)
```
Then those msq queue will be processing in the Parser

Reply:
Yes, when we receive a message it gets passed to the parser. If it's not valid "KERI" with the correct signatures etc, it won't complete parsing.

In your example, there's just one msg though. The attachments is just supporting info for `serder`

Reply:
not sure I got your mean, but I see that we cloned the code from here:
https://github.com/WebOfTrust/keripy/blob/main/src/keri/app/indirecting.py#L901
then we only customized to check and validate REGISTRAR_SEAL_SAID for icp event message. 
So I prefer to keep current logic and only change 
`if serder.ked["t"] == "icp" `
to
```
ilk = serder.ked["t"]
if ilk in (Ilks.icp,)
```

Reply:
@Sotatek-Patrick-Vu To be clear, the cardano-backer MUST provide receipts for `icp='icp', rot='rot', ixn='ixn', dip='dip', drt='drt'`.

As far as I know (I need to confirm), the `REGISTRAR_SEAL_SAID` can appear in `icp` or `rot`.

My goal is this:
- If an identifier is incepted (icp) on Cardano with `REGISTRAR_SEAL_SAID`, we write it on-chain
- If an identifier is incepted using witnesses, but later changes to Cardano with a `rot` and `REGISTRAR_SEAL_SAID`, we write it on-chain (including the icp)
- If an identifier _was_ on Cardano, but then decides with a `rot` to switch back to witnesses, we should stop writing it on-chain

We might also in the future write witness KELs on-chain as a backup mechanism, but not right now